{"uuid": "46b58197-f6c7-4035-bbaf-dd95f6ed5154", "children": ["8b6b11cf-82bf-4827-9d43-5f8087304ee8"], "befores": [{"name": "page", "status": "passed", "start": 1759400724521, "stop": 1759400724715}], "afters": [{"name": "page::0", "status": "broken", "statusDetails": {"message": "FileNotFoundError: [WinError 2] The system cannot find the file specified\n", "trace": "  File \"C:\\Users\\newaj\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\allure_commons\\_allure.py\", line 231, in __call__\n    return self._fixture_function(*args, **kwargs)\n           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\newaj\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\_pytest\\fixtures.py\", line 938, in _teardown_yield_fixture\n    next(it)\n    ~~~~^^^^\n  File \"C:\\Users\\newaj\\PycharmProjects\\Magento-Demo-Site-E2E-Automation\\tests\\conftest.py\", line 33, in page\n    allure.attach.file(\n    ~~~~~~~~~~~~~~~~~~^\n        video_path,\n        ^^^^^^^^^^^\n        name=\"Failure Video\",\n        ^^^^^^^^^^^^^^^^^^^^^\n        attachment_type=allure.attachment_type.WEBM\n        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n    )\n    ^\n  File \"C:\\Users\\newaj\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\allure_commons\\_allure.py\", line 213, in file\n    plugin_manager.hook.attach_file(source=source, name=name, attachment_type=attachment_type, extension=extension)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\newaj\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\pluggy\\_hooks.py\", line 512, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n           ~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\newaj\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\pluggy\\_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n           ~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\newaj\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\pluggy\\_callers.py\", line 167, in _multicall\n    raise exception\n  File \"C:\\Users\\newaj\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\pluggy\\_callers.py\", line 121, in _multicall\n    res = hook_impl.function(*args)\n  File \"C:\\Users\\newaj\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\allure_pytest\\listener.py\", line 258, in attach_file\n    self.allure_logger.attach_file(uuid4(), source, name=name, attachment_type=attachment_type, extension=extension)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\newaj\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\allure_commons\\reporter.py\", line 160, in attach_file\n    plugin_manager.hook.report_attached_file(source=source, file_name=file_name)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\newaj\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\pluggy\\_hooks.py\", line 512, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n           ~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\newaj\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\pluggy\\_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n           ~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\newaj\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\pluggy\\_callers.py\", line 167, in _multicall\n    raise exception\n  File \"C:\\Users\\newaj\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\pluggy\\_callers.py\", line 121, in _multicall\n    res = hook_impl.function(*args)\n  File \"C:\\Users\\newaj\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\allure_commons\\logger.py\", line 39, in report_attached_file\n    shutil.copy2(source, destination)\n    ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\newaj\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\shutil.py\", line 453, in copy2\n    _winapi.CopyFile2(src_, dst_, flags)\n    ~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^\n"}, "attachments": [{"name": "Failure Screenshot", "source": "a1b1bcb3-5985-4eac-aa6b-1e69d891a973-attachment.png", "type": "image/png"}, {"name": "Failure Video", "source": "9497e0a2-ef46-4e79-8482-5e81ce180a66-attachment.webm", "type": "video/webm"}], "start": 1759400724722, "stop": 1759400725118}], "start": 1759400724521, "stop": 1759400725118}